Closes #4

vfournier11 opened the pull request.

Commit:
Add simple Docker-compose configuration

- Add ExpressJs application with reverse proxy to the server
- Add Flask server
- Create Docker-compose to link them together
- First draft of the development readme

Commit:
Fix links within the README

Commit:
Add image processing engine (ipengine)

- Link Flask container with ipengine
- Add script to compile within the ipengine Docker container to compile the c++ in Ubuntu
- Create simple c++ program that log args

vfournier11 commented:
Could you test that and review the process.
Don't hesitate to ask question!
Everything should work in Windows, MacOs

renaudfv commented:
First try yields : ERROR: Couldn't connect to Docker daemon - you might need to run `docker-machine start default`.

vfournier11 committed:
Fix problems when mounting web-client volume

- Now only mount public/ and server.js
- Add first compilation of ipengine

Commit:
Add config for local development of web-client

- Change default port to 6088 for web-client
- Now mount all web-client/ folders when using docker-compose.dev.yml
